models/swapr/ethereum/swapr_ethereum_trades.sql

```diff
@@ -53,9 +53,9 @@ SELECT
     end as token_pair,
     dexs.token_bought_amount_raw / power(10, erc20a.decimals) AS token_bought_amount,
     dexs.token_sold_amount_raw / power(10, erc20b.decimals) AS token_sold_amount,
-    dexs.token_bought_amount_raw,
-    dexs.token_sold_amount_raw,
-    coalesce(
+    CAST(dexs.token_bought_amount_raw AS DECIMAL(38,0)) AS token_bought_amount_raw,
+    CAST(dexs.token_sold_amount_raw  AS DECIMAL(38,0)) AS token_sold_amount_raw,
+    coalesce(   
```

Comment:
nit: whitespace

Reply:
I'm splitting out these changes to only cover uniswap for now, as per Thomas' comment - but definitely, I'll fix this in a future PR, thank you!